src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPCompletionFeature.java

```diff
@@ -220,27 +220,38 @@ public boolean isItemTextBold(@NotNull CompletionItem item) {
      * @param lookupItem
      * @param priority
      * @param item
+     * @param caseSensitive
      */
     @ApiStatus.Internal
     public void addLookupItem(@NotNull CompletionPrefix completionPrefix,
                               @NotNull CompletionResultSet result,
                               @NotNull LookupElement lookupItem,
                               int priority,
-                              @NotNull CompletionItem item) {
+                              @NotNull CompletionItem item,
+                              boolean caseSensitive) {

```

Comment:
Even if it is internal, please try to avoid updating the signature. The caseSensitive can be get inside the method.

Reply:
I did look at doing exactly that but wasn't sure how to get the language server definition -- or at least the language ID -- from the existing parameters that are passed into `addLookupItem()`. I can get the PSI element from the lookup item. Is there a santioned way to get the language server definition/language ID given only a PSI element? I saw `LanguageServerRegistry#findLanguageServerDefinitionFor(Language, FileType, VirtualFile)` and `LanguageServerAccessor#getMatchedLanguageServerDefinitions()`, but those both seem internal right now as well.